lib/h_pyramid_sentry/README.md

```diff
@@ -0,0 +1,47 @@
+Hypothesis Pyramid Sentry Extension
+===================================
+
+A small library intended to log messages to help [Sentry](https://sentry.io)
+with the ability to filter out unwanted messages.
```

Comment:
I think what this extension actually does is add Sentry support to your Pyramid app (it's the extension that depends on `sentry_sdk` and calls `sentry_sdk.init()`, the app wouldn't be reporting to Sentry at all without the `h_pyramid_sentry` extension). Plus also it adds some custom h customisation and features on top of sentry_sdk: it initialises sentry_sdk with the Pyramid and Celery integration; adds the filter functions things; and adds the `pyramid_retry` support).

Reply:
This was fixed up in the new repo.